infra/src/dsl/xroad.ts

```diff
@@ -312,3 +312,13 @@ export const FishingLicense = new XroadConf({
     },
   },
 })
+
+export const Vehicles = new XroadConf({
+  env: {
+    VEHICLES_XROAD_PATH: {
```

Comment:
yeah, this is kind of a nitpick but all the xroad values use the `XROAD_` [as prefix](https://github.com/island-is/island.is/pull/7144/files#diff-49a22b24f1b88c92b9b5cb48c9390192375b407fe22fa373db9fda364c2ad4ddR271)

Reply:
✅